.dependabot/config.yml

```diff
@@ -18,20 +18,20 @@ update_configs:
       # objects which do not work in IE11
       - match:
           dependency_name: "ember-changeset"
-          version_requirement: ">=3"
+          version_requirement: ">= 3"
       - match:
           dependency_name: "ember-changeset-validations"
-          version_requirement: ">=3"
+          version_requirement: ">= 3"
 
       # graphql-tools v5+ breaks generic mocking
```

Comment:
this is not correct anymore, is it?

Reply:
Sure it is, we still have to support IE11 which sadly didn't magically develop the ability to use `Proxy` :wink:

Reply:
I'm confused because the comment says "graphql-tools v5+ breaks generic mocking" and the version requirement was changed from "<=4" to ">= 5"

Reply:
As discussed directly: the previous configuration was wrong since it's an **ignore** configuration. We want dependabot to ignore v5+